x-pack/plugins/transform/server/routes/api/privileges.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  APP_CLUSTER_PRIVILEGES,
+  APP_INDEX_PRIVILEGES,
+} from '../../../../../legacy/plugins/transform/common/constants';
+// NOTE: now we import it from our "public" folder, but when the Authorisation lib
+// will move to the "es_ui_shared" plugin, it will be imported from its "static" folder
+import { Privileges } from '../../../../../legacy/plugins/transform/public/app/lib/authorization';
```

Comment:
@sebelga I originally copied this from the `snapshot_restore` plugin in around October 2019. In the meantime is there something available from `es_eui_shared`? Or is it fine to just remove the comment and leave the code as is?

Reply:
You can remove the comment. I'd like to get to it at some point but might not be soon.